File: src/app/model/item-venda-acessorio.ts
export class ItemVendaAcessorio {
    id!: number;
    valor!: string;
    quantidade!: number;
    iditemVenda!: number;
    idAcessorio!: number;


    constructor(){
        this.id = 0;        
        this.valor = "";
        this.quantidade = 0;
        this.iditemVenda = 0;
        this.idAcessorio = 0;
    }
}

File: api/src/main/java/br/cefet/vidralglass/model/ItemVendaAcessorio.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package br.cefet.vidralglass.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
import lombok.experimental.SuperBuilder;

/**
 *
 * @author criad
 */
@Data
@Getter
@Setter
@ToString
@SuperBuilder
@NoArgsConstructor
@AllArgsConstructor
public class ItemOrcamentoAcessorio {
    protected int idItemOrcamentoAcessorio;
    protected int valor;
    protected int quantidade;
    protected int idItemOrcamento;
    protected int idAcessorio;
    protected int valorTotal;
}
